packages/cf-component-box/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cf-component-box",
   "description": "Cloudflare Box Component",
-  "version": "1.4.3",
+  "version": "2.0.0",
```

Comment:
somehow @koddsson you published version 2.0.0 of the package from this branch. I strongly disagree with publishing other than master branch. Our experience dealing with `legacy` branch showed that Lerna is not capable of resolving it properly.